gadget/layout.go

```diff
@@ -25,91 +25,91 @@ import (
 	"sort"
 )
 
-// PositioningConstraints defines the constraints for positioning structures
-// within a volume
-type PositioningConstraints struct {
+// LayoutConstraints defines the constraints for arranging structures within a
+// volume
+type LayoutConstraints struct {
 	// NonMBRStartOffset is the default start offset of non-MBR structure in
 	// the volume.
 	NonMBRStartOffset Size
 	// SectorSize is the size of the sector to be used for calculations
 	SectorSize Size
 }
 
-// PositionedVolume defines the size of a volume and positions of all the
+// LaidOutVolume defines the size of a volume and arrangement of all the
 // structures within it
-type PositionedVolume struct {
+type LaidOutVolume struct {
 	*Volume
 	// Size is the total size of the volume
 	Size Size
 	// SectorSize sector size of the volume
 	SectorSize Size
-	// PositionedStructure are sorted in order of 'appearance' in the volume
-	PositionedStructure []PositionedStructure
+	// StructureLayout are the structures within the volume, sorted by their
+	// start offsets
+	StructureLayout []LaidOutStructure
```

Comment:
If we followed the pattern this would have been: LaidOutStructure.   We can probably call this just Layout though, no?

Reply:
Reamed to LaidOutStructure.

Reply:
As I understand it (per comment below re LaidOutContent) this structure also becomes `LaidOut...` rather than a plain `Layout`, for consistence?